<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSearchOrderForm</class>
 <widget class="QWidget" name="AdminSearchOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>甜鱼: 管理员搜索订单界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    border: none;
    border-radius: 4px;
	padding: 4px;
	margin: 4px; 
    background-color: rgb(54, 125, 221);
    color: white;
}
QLineEdit{
	border: 1px solid black;
	border-radius: 4px;
	padding: 4px;
}
</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>747</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QRadioButton" name="check_buyer_uid">
      <property name="text">
       <string>买家UID</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="check_oid">
      <property name="text">
       <string>OID</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索内容:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>甜鱼二手交易平台</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>搜索关键字:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>管理员搜索订单界面</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QRadioButton" name="check_seller_uid">
      <property name="text">
       <string>卖家UID</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="search_content_text"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="postBtn">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
